.github/workflows/Build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on: [push, pull_request]
+jobs:
+  AngouriMath:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - name: Setup .NET 5
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.100-rc.1.20452.10'
+    - name: Build
+      run: dotnet build AngouriMath
+  Everything:
+    runs-on: windows-latest  # Window application cannot be run on other OS
```

Comment:
```suggestion
    runs-on: windows-latest  # Windows application cannot be run on other OS
```

Reply:
That's super minor tbh

Reply:
What is a Window application? You apply a window on some other object?